shipyard.md

```diff
@@ -11,27 +11,44 @@ A shipyard file can consist of any number of stages. A stage has the following p
 
 * **Name**. The name of the stage. This name will be used for the GitOps branch and the Kubernetes namespace to which services at this stage will be deployed to. Future versions will allow to explicitly define the target namespace.
 
-* **Deployment Strategy**. The deployment strategy used to deploy a new version of a service. Keptn supports deployment strategies of type: 
+* **Deployment Strategy**. The deployment strategy defines how an artifact is deployed. Currently, the following strategies are supported: 
   * `direct`: Deploys a new version of a service by replacing the old version of the service.
   * `blue_green_service`: Deploys a new version of a service next to the old one. After a successful validation of this new version, it replaces the old one and is marked as stable (i.e. it becomes the `primary`-version).
 
-   Future versions of Keptn will also support canary deployments.
+   Future versions of Keptn will also support canary releases.
 
-* **Test Strategy**. Defines the test strategy used to validate a deployment. Failed tests result in an automatic roll-back of the latest deployment in case of a blue/green deployment strategy. Keptn supports tests of type:
+* **Test Strategy**. Defines the test strategy used to validate a deployment. Failed tests result in an automatic roll-back of the latest deployment in case of a blue/green deployment strategy is used. Keptn supports tests of type:
   * `functional` 
   * `performance` 
 
+  In future versions of Keptn, it is not necessary to distinguish between functional and performance tests. But it is necessary to define multiple tests for a stage that can either be executed sequentially or in parallel. Therefore, we need a link between the shipyard and the uniform (i.e. using selectors and labels as known by k8s).
+
+* **Remediation Strategy**. The remediation strategy specifies whether remediation actions should automatically be executed in the respective stage. If Keptn receives a problem event (of type `sh.keptn.event.problem.open`), a remediation action is defined for this problem (in the `remediation.yaml` file), and the remediation is set to `automated`, then the remediation action is executed. 
+Keptn supports remediations of type:
+  * `automated`
+  
+* **Approval Strategy**. The approval strategy specifies the kind of approval before promoting an artifact into the next stage. Additionally, it specifies the evaluation result (i.e. can be `pass`, `warning`, and `fail`) that needs to be passed for promoting the artifact. 
+Keptn will/should support the following approval strategies:
```

Comment:
How would one define an approval strategy where the evaluation result "pass" is promoted automatically and "warning"/"fail" manually? Should we consider configuring the approval strategy for each evaluation result individually?

Reply:
This is exactly the use-case we discussed in the last community meeting. One idea was to separate the quality criteria of the approval *behavior*:

Proposal: 
```
approval_strategy: 
- pass: automatic
- warning: manual
- fail: manual
```
Is there a disadvantage/limitation of my proposal?

Reply:
Can we extend this by making "automatic" the default option so this parameter can be omitted? Conversely, that would mean one would only specify an approval_strategy explicitly if it is manual, correct?
Also, why would I want to manually approve a failed evaluation? It most likely failed for a reason - do we really want this as an option?

Reply:
@dirkwall How would your suggestion of a `approval_strategy` with `automatic` as default look like? Similar like below?
```
approval_strategy: 
- warning
- fail
```

I think the option of manually approve a failed evaluation makes sense in case a user wants to overrule the evaluation result. Of course, the user needs to provide an explanation why a failed evaluation should be promoted.